docs/configuring-crash-stats.rst

```diff
@@ -0,0 +1,374 @@
+.. index:: configuring-crashstats
+
+.. _configuring-crashstats-chapter:
+
+Configuring Crash-Stats and PostgreSQL
+======================================
+
+https://crash-stats.mozilla.org is Mozilla's front-end to expose crash-stats
+to developers and interested users. It is written with Mozilla's unique 
+requirements in mind, and will probably not be useful to other Socorro users.
+
+You probably do not want to use this, unless you are Mozilla.
+
+Install PostgreSQL
+------------------
+
+Install the PostgreSQL repository.
+::
+  sudo rpm -ivh http://yum.postgresql.org/9.3/redhat/rhel-7-x86_64/pgdg-centos93-9.3-1.noarch.rpm
+
+Now you can actually install the packages:
+::
+  sudo yum install postgresql93-server postgresql93-contrib
+
+Initialize and enable PostgreSQL on startup:
+::
+  sudo service postgresql-9.3 initdb
+  sudo systemctl enable postgresql-9.3
+
+Modify postgresql config
+::
+  sudo vi /var/lib/pgsql/9.3/data/postgresql.conf
+
+Ensure that timezone is set to UTC
+::
+  timezone = 'UTC'
+
+Allow local connections for PostgreSQL
+::
+  sudo vi /var/lib/pgsql/9.3/data/pg_hba.conf
+
+Ensure that local connections are allowed:
+::
+
+  # IPv4 local connections:
+  host    all             all             127.0.0.1/32            trust
+  # IPv6 local connections:
+  host    all             all             ::1/128                 trust
+
+See http://www.postgresql.org/docs/9.3/static/auth-pg-hba-conf.html
+for more information on this file.
+
+You'll need to restart postgresql if the configuration was updated:
+::
+  sudo systemctl restart postgresql-9.3
+
+Create database, set up schema
+------------------------------
+
+Socorro provides a setup script which attempts to initialize PostgreSQL::
+
+    sudo setup-socorro.sh postgresql
+
+This creates a new database named "breakpad" and sets up the schema
+that Socorro and crash-stats expect.
+
+Configure Socorro Processor
+---------------------------
+
+Socorro Processor must be configured to store crashes in Elasticsearch
+as well as PostgreSQL.
+
+NOTE - you must replace @@@DATABASE_HOSTNAME@@@, @@@DATABASE_USERNAME@@@ and
+@@@DATABASE_PASSWORD@@@ below with valid entries for your PostgreSQL install
```

Comment:
I don't know what these `@@@` are but if you want to make the strings monospace you can just use a double back-tick.